src/components/BadgerButton/stories.js

```diff
@@ -294,8 +294,33 @@ storiesOf('BadgerButton', module)
 			<BadgerButton
 				paymentRequestUrl={text(
 					'Invoice URL',
-					'https://yourInvoiceUrlHere.com/String'
+					// expired invoice
```

Comment:
Rename this story to `BIP70 - expired invoice` or something more descriptive

Reply:
d5124c4